e2e/login.e2e-spec.ts

```diff
@@ -0,0 +1,128 @@
+import type { INestApplication } from '@nestjs/common';
+import { HttpStatus } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import * as pactum from 'pactum';
+
+import { adminStub, loginInputValidPasswordDtoStub } from '../src/admin/stubs';
+import { AppModule } from '../src/app.module';
+import { PrismaService } from '../src/prisma/prisma.service';
+
+function graphqlRequest(query: string) {
+  return pactum
+    .spec()
+    .post('/graphql')
+    .withHeaders('Content-Type', 'application/json')
+    .withGraphQLQuery(query)
+    .expectStatus(HttpStatus.OK);
+}
+
+describe('Login', () => {
+  let app: INestApplication;
+  let prisma: PrismaService;
+
+  beforeAll(async () => {
+    const moduleRef = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication();
+
+    prisma = app.get(PrismaService);
+
+    await prisma.admin.deleteMany();
```

Comment:
this is not a good idea. because each time we run the test in local environment our admin users will be deleted completely.

you can create a fake admin in start of test and remove it at the end to make test idempotent.

Reply:
it's in test mode,

Reply:
what is the value of cleaning database at test start here? I can't think of one 🤔

Reply:
is there a better way to do that? I do it for more confidence

Reply:
just don't do that :)

Reply:
ok i will find better why, but for now it's ok :+1: